src/main/resources/application-dev.yml

```diff
@@ -36,3 +36,13 @@ logging:
 
 server:
   shutdown: immediate
+
+offender-events:
+  sqs-queue: https://sqs.eu-west-2.amazonaws.com/754256621582/Digital-Prison-Services-dev-hmpps_tier_offender_events_queue
```

Comment:
this isn't a _secret_ but I'd still put it in secrets manager, I can demonstrate.